docs/migration/migration.md

```diff
@@ -0,0 +1,147 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "Terraform Migration Guide"
+subcategory: ""
+description: |-
+  
+---
+
+# Terraform Migration Guide
+
+## Introduction
+This document provides critical steps to migration your Terraform deployment for BloxOne DDI.
+
+## Simplified Steps
+
+- `Backup` your existing Terraform state file
+- `List all resources` in your existing Terraform state file
+- `Clean up old provider state file` by removing all resources from the state file
+- `Update all resources` in your existing Terraform code
+- `Run Terraform plan` to verify the changes
+
+## Detailed Steps
+## Backup
```

Comment:
To make it hierarchical , '###' subheading can be used for this and all subsequent blocks.
If it makes the size of text too small , ignore this comment.
```suggestion
### Backup
```